app/controllers/users/phone_confirmation_controller.rb

```diff
@@ -48,5 +54,9 @@ def after_confirmation_path
         after_sign_in_path_for(current_user)
       end
     end
+
+    def check_for_unconfirmed_phone
+      redirect_to root_path unless unconfirmed_phone
+    end
```

Comment:
This is a duplication of the same method that is already included via `PhoneConfirmationFlow`.